plat/xen/arm/setup64.c

```diff
@@ -358,9 +358,5 @@ void _libxenplat_armentry(void *dtb_pointer, paddr_t physical_offset)
 	get_console();
 	get_xenbus();
 
-	prepare_console();
```

Comment:
In order for this to work, besides these changes you need to move `uk_boot_early_init()` after `get_console()` and `get_xenbus()`, as these populate `HYPERVISOR_start_info`.

Reply:
Thank you for figuring this out Michalis! I updated the commit.